gapidapk/android/apk/AndroidManifest.xml.in

```diff
@@ -24,6 +24,9 @@
         android:minSdkVersion="21"
         android:targetSdkVersion="26" />
 
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Neither of these should be required. If the cube needs them, it should be changed to not need them.